cdash/nightly.sh

```diff
@@ -20,13 +19,13 @@ function getname() {
 export engpar=$root/`getname engpar`/install # This is where engpar will be (or is) installed
 export kk=$root/`getname kokkos`/install   # This is where kokkos will be (or is) installed
 export oh=$root/`getname omegah`/install  # This is where omega_h will be (or is) installed
-export oh1050=$root/`getname omegah1050`/install
 export pumipic=$root/`getname pumipic`/install # This is where PumiPIC will be (or is) installed
 export CMAKE_PREFIX_PATH=$engpar:$kk:$oh:$pumipic:$CMAKE_PREFIX_PATH
 export MPICH_CXX=$root/kokkos/bin/nvcc_wrapper
 
 cd $root
-[ ! -d kokkos ] && git clone -b 3.4.01 git@github.com:kokkos/kokkos.git
+[ ! -d kokkos ] && git clone git@github.com:kokkos/kokkos.git
```

Comment:
This `[ ! -d kokkos ] && git clone -b 4.0.01 git@github.com:kokkos/kokkos.git`
should avoid having to do the the seperate checkout, pull, and cd commands.

Reply:
Yeah that seems better. I made the change.